nc models/app.model.js

```diff
@@ -6,3 +6,11 @@ exports.allTopics = () => {
     return rows;
   });
 };
+
+exports.allArticles = (article_id) => {
```

Comment:
when naming functions try to be as explicit as you can in what the function is doing, in this example you have named allArticles when in practice you are only fetching one article by its article ID